### PR TITLE
[ODS-4972] Make swagger year selection configurable

### PR DESCRIPTION
**Pull request by tnunnerydl**

### CODE REVIEW

File: Application/EdFi.Ods.SwaggerUI/appsettings.json [hunk 1]
```diff
@@ -6,6 +6,18 @@
...
+      "Year": "2020"
+    },
+    {
+      "Year": "2021",
+      "IsDefault": true
+    },
+    {
+      "Year": "2022"
+    }
+  ],
```
Comment (tbmeador-dlp) [inline Application/EdFi.Ods.SwaggerUI/appsettings.json:20]: Would it be a good idea and remove this so we can just use the defaults? that way we don't have to update this every year. cc @vimayya

**Review by tbmeador-dlp — CHANGES_REQUESTED** [review]
testing looks good, a little bit of cleanup needed

File: Application/EdFi.Ods.WebApi/appsettings.json [hunk 2]
```diff
@@ -16,6 +16,19 @@
...
+      },
+      {
+        "Year": "2021",
+        "IsDefault": true
+      },
+      {
+        "Year": "2022"
+
+      }
+    ],
```
[the remark this answers is not in the document]
Comment (tnunnerydl) [reply Application/EdFi.Ods.WebApi/appsettings.json]: Done.

File: Application/EdFi.Ods.SwaggerUI/appsettings.json (hunk 1 above)
Comment (tnunnerydl) [reply Application/EdFi.Ods.SwaggerUI/appsettings.json]: Done.

**Review by tbmeador-dlp — APPROVED** [review]

File: Application/EdFi.Ods.SwaggerUI/appsettings.json (hunk 1 above)
Comment (vimayya) [reply Application/EdFi.Ods.SwaggerUI/appsettings.json]: sure. I will add it to documentation.

**Pull request merged**